packages/version-bump/src/main.ts

```diff
@@ -124,7 +103,7 @@ export async function run(): Promise<void> {
 		}
 
 		// if version type is decaf then let's update extra values in main file and the readme.txt as well.
-		if (type === 'decaf') {
+		if (releaseType === 'decaf') {
 			// but we're also changing the plugin name and uri
 			const pluginUri = mainFileContents.match(MAIN_FILE_PLUGIN_URI_REGEX)?.groups?.plugin_uri?.trim();
```

Comment:
we should move all of this file rewriting logic into its own file/script/action
but that can be done in another PR

Reply:
Yeah, we can do that, I have created this `io` packages which can be used for such things.